src/main/resources/site/parts/statistics/statistics.es6

```diff
@@ -23,8 +26,16 @@ exports.preview = (req) => renderPart(req)
 const renderPart = (req) => {
   const page = getContent()
 
+  log.info(`page info ${JSON.stringify(page.data)}`)
+  const statistic = page.data.statistic && getStatisticByIdFromRepo(page.data.statistic)
+  const publications = statistic && getPublicationsForStatistic(statistic.shortName)
+
```

Comment:
Finnes ikke noe felt shortName på statistikk, bør også ha en sjekk på om shortName finnes. Slik koden er nå er det en Feil i visning av statistikk siden shortName er undefined

Reply:
I was not able to reproduce this. By any chance do you have a statistikk that gave you an error here?